internal/api/req.go

```diff
@@ -167,6 +169,7 @@ func (r *Request) HTTPSDownload(filepath, friendlyName string) (*RawResponse, er
 	}
 
 	req.Header.Add("content-type", "application/json")
+	req.Header.Add("user-agent", fmt.Sprintf("ironstar-cli/%s", version))
```

Comment:
I understand it's not part of your changes, but ln. 76 (and in other HTTP functions also) `InsecureSkipVerify` probably should be `false`. I don't remember why I did this, likely for local testing against Nankai which has a self-signed cert

I don't particularly want to break that, although I haven't used it for a while. Perhaps the condition for `InsecureSkipVerify` could be set as whether the `IRONSTAR_API_ADDRESS` is set to override from api.ironstar.io. Would need to take care when testing stage though as the override is needed to target

Reply:
Nankai on our local machines uses a self-signed cert. So would break local development. There could be an explicit env var to set that would override this for local dev potentially. Or at least a warning somewhere so the next person understands why local dev isn't working.